tests/test_addresses.py

```diff
@@ -76,10 +105,36 @@ def test_deployments_invalid_format():
             }
         }
     )
+    responses.add(
```

Comment:
Nicely done, but please pay attention to the name of the test.

It's always good to make tests as small as possible.

My suggestion: create separate test functions for both extras and msigs logic

Reply:
I tried that but you have to copy all the inputs each time and then there is very little additional code.  It felt silly and made changes to the test inputs  hard.

Reply:
You should separate all the test cases in smaller units, that is my recommendation. It's better to NOT test everything in one function even if it leads to boilerplate code

Reply:
Another day ser :)